tensorboard/plugins/graph/tf_graph_dashboard/tf-graph-dashboard.html

```diff
@@ -324,25 +328,38 @@ <h3>No graph definition files were found.</h3>
 
     // Set this to true so we only initialize once.
     this._initialized = true;
-    Promise.all([this._fetchGraphRuns(), this._fetchRunMetadataTags()])
-      .then(result => {
-        const runsWithGraph = result[0].sort(vz_sorting.compareTagNames);
-        const runToMetadata = result[1];
-        const datasets = _.map(runsWithGraph, runName => ({
-          name: runName,
-          path: this._graphUrl(
-              runName, tf.graph.LIMIT_ATTR_SIZE, tf.graph.LARGE_ATTRS_KEY),
-          runMetadata: _.map(
-            (runToMetadata[runName] || []).sort(vz_sorting.compareTagNames),
-            tag => ({
-              tag: tag,
-              path: tf_backend.addParams(
-                tf_backend.getRouter().pluginRoute('graphs', '/run_metadata'),
-                {tag: tag, run: runName}),
-            })),
-        }));
-        this.set('_datasets', datasets);
+    this._fetchDataset().then(dataset => {
+      const runNames = Object.keys(dataset);
+      // Transform raw data into UI friendly data.
+      this._datasets = runNames.sort(vz_sorting.compareTagNames).map(runName => {
+        const runData = dataset[runName];
+
+        const tagNames = Object.keys(runData.tags).sort(vz_sorting.compareTagNames);
+        const tags = tagNames
+            .map(name => runData.tags[name])
+            .map(({tag, conceptual_graph, op_graph, profile}) => ({
+              tag,
+              tagName: tag,
+              conceptualGraph: conceptual_graph,
+              opGraph: op_graph,
+              profile,
+            }));
+
+        const tagsWithV1Graph = runData.run_graph ?
+            [
+                {
+                  tag: null,
+                  tagName: 'Default',
+                  conceptualGraph: false,
+                  opGraph: true,
```

Comment:
Could you please add a comment here to call attention to the fact that
we’re projecting a run graph down to a specially named op graph so that
the future components of the pipeline don’t have to care about run
graphs? It’s a great idea, but it wasn’t obvious to me what the
implications were; when reading `tf-graph-loader._load()` I was confused
because I didn’t see it explicitly loading any run graphs.